app/scripts/views/mixins/notifications-mixin.js

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A view mixin to interface with the notifications channel
+ *
+ * Any registered handlers are automatically unregistered
+ * when the view is destroyed.
+ */
+
+define([
+], function () {
+  'use strict';
+
+  var NotificationsChannelMixin = {
+    notificationsInitialize: function (options) {
```

Comment:
:+1: Maybe instead of using `notificationsOn`, `notificationsOff`, etc, a consuming module would still call `this.notifications.on`, but notifications would not be the passed in notifications object, but rather a proxy that does the housekeeping embedded here. Thoughts?